src/hooks/useTabVisibility.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react'
+
+const useTabVisibility = () => {
```

Comment:
this is not used and logic moved to useTabAcitivity hook but for now I keep it because it might be used if this feature implementation changes